### ISSUE TITLE
Positive calf squeeze test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11158]
**Preferred term label:**
Positive calf squeeze test

**Synonyms**
Simmonds-Thompson Test
Thompson Test

**Definition (free text, please give PubMed ID)**
With the patient laying prone and feet hanging over the edge of a table, passive squeeze of the calf does not reproduce normal plantarflexion of the foot

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Achilles tendon rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001432
- label: Positive calf squeeze test
- parents: [Abnormal musculoskeletal physiology (HP:0011843)](https://hpo.jax.org/app/browse/term/HP:0011843)
- pmid: PMID:25409728
- def: An abnormal result of the calf-squeeze test (also known as the Thompson test), which is considered a sign of Achilles tendon rupture. The test is positive (abnormal) if no plantar flexion of the foot occurs at calf squeezing.
- comment: With the patient laying prone and feet hanging over the edge of a table, passive squeeze of the calf does not reproduce normal plantarflexion of the foot.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11158
- synonyms: Positive Thompson test(NONE)|Positive Simmonds-Thompson test(NONE)
